app/src/main/java/com/cornellappdev/uplift/ui/screens/SignInPromptScreen.kt

```diff
@@ -0,0 +1,229 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.util.GRAY01
+import com.cornellappdev.uplift.util.GRAY02
+import com.cornellappdev.uplift.util.GRAY04
+import com.cornellappdev.uplift.util.LIGHT_YELLOW
+import com.cornellappdev.uplift.util.PRIMARY_YELLOW
+import com.cornellappdev.uplift.util.PRIMARY_YELLOW_BACKGROUND
+import com.cornellappdev.uplift.util.montserratFamily
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.systemuicontroller.SystemUiController
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+@OptIn(ExperimentalPermissionsApi::class)
+@Composable
+fun SignInPromptScreen() {
+    val systemUiController: SystemUiController = rememberSystemUiController()
+
+    systemUiController.isStatusBarVisible = false
+    systemUiController.isNavigationBarVisible = false // Navigation bar
+    systemUiController.isSystemBarsVisible = false
+
+    Box() {
+        Canvas(modifier = Modifier.fillMaxSize()) {
+            drawCircle(
+                color = LIGHT_YELLOW,
+                radius = 244.dp.toPx(),
+                center = Offset(x = 142.dp.toPx(), y = (-35).dp.toPx())
+            )
+        }
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier.fillMaxSize()
+        ) {
+            Spacer(modifier = Modifier.height(88.dp))
+
+            Image(
+                painter = painterResource(id = R.drawable.ic_main_logo),
+                contentDescription = "Uplift logo",
+                modifier = Modifier
+                    .height(130.dp)
+                    .width(115.dp)
+            )
+            Spacer(modifier = Modifier.height(60.dp))
+
+            Text(
+                text = "Find what uplifts you.",
+                fontSize = 24.sp,
+                fontFamily = montserratFamily,
+                fontWeight = FontWeight.Bold
+            )
+
+            Spacer(modifier = Modifier.height(80.dp))
+
+            Text(
+                "Log in to:",
+                fontSize = 16.sp,
+                fontFamily = montserratFamily,
+                fontWeight = FontWeight.Bold
+            )
+
+            Spacer(modifier = Modifier.height(20.dp))
+
+            upliftUsesCardList()
+
+            Spacer(modifier = Modifier.height(125.dp))
+            signinButton()
+
+
+        }
+    }
+}
+
+@Composable
+private fun signinButton() {
```

Comment:
`SignInButton` plz rename,

Also there are some `onClick` functions here that currently can't do anything because you're taking nothing in through parameters. Take in an `onClick` as a parameter so this component can be meaningfully used.